src/Tizen.Multimedia.Camera/Camera/Camera.cs

```diff
@@ -50,6 +50,8 @@ public class Camera : IDisposable, IDisplayable<CameraError>
         /// Initializes a new instance of the <see cref="Camera"/> class.
         /// </summary>
         /// <param name="device">The camera device to access.</param>
+        /// <exception cref="InvalidOperationException">Invalid CameraDevice type.</exception>
```

Comment:
According to description, It seem to ArgumentException,

Reply:
You're right. I'll fix it.